ingestion/src/metadata/cli/ingest_dbt.py

```diff
@@ -0,0 +1,210 @@
+        missing_vars = []
+        if not host_port:
+            missing_vars.append('openmetadata_host_port')
+        if not jwt_token:
+            missing_vars.append('openmetadata_jwt_token')
+        if not service_name:
+            missing_vars.append('openmetadata_service_name')
+            
+        raise ValueError(
+            f"Required OpenMetadata configuration not found in dbt_project.yml vars.\n"
+            f"Missing variables: {', '.join(missing_vars)}\n"
+            f"Please add the following to your dbt_project.yml:\n"
+            f"vars:\n"
+            f"  openmetadata_jwt_token: 'your-jwt-token'\n"
+            f"  openmetadata_host_port: 'your-host-port (e.g. http://openmetadata-server:8585/api)'\n"
+            f"  openmetadata_service_name: 'your-service-name'"
+        )
+    
+    return {
+        'jwt_token': jwt_token,
+        'host_port': host_port,
+        'service_name': service_name
+    }
+
+
+def create_dbt_workflow_config(dbt_project_path: Path, om_config: Dict) -> Dict:
+    """
+    Create OpenMetadata workflow configuration for dbt artifacts ingestion
+    
+    :param dbt_project_path: Path to the dbt project directory
+    :param om_config: OpenMetadata configuration
+    :return: Workflow configuration
+    """
+    target_dir = dbt_project_path / "target"
+    
+    # Check for required artifacts
+    manifest_path = target_dir / "manifest.json"
+    if not manifest_path.exists():
+        raise FileNotFoundError(
+            f"manifest.json not found in {target_dir}. Please run 'dbt compile' or 'dbt run' first."
+        )
+    
+    # Build dbt config source
+    dbt_config_source = {
+        'dbtManifestFilePath': str(manifest_path),
+        'dbtConfigType': 'local'
+    }
+    
+    # Add optional files if they exist
+    catalog_path = target_dir / "catalog.json"
+    if catalog_path.exists():
+        dbt_config_source['dbtCatalogFilePath'] = str(catalog_path)
+    
+    run_results_path = target_dir / "run_results.json"
+    if run_results_path.exists():
+        dbt_config_source['dbtRunResultsFilePath'] = str(run_results_path)
+    
+    # Create workflow configuration
+    config = {
+        'source': {
+            'type': 'dbt',
+            'serviceName': om_config['service_name'],
+            'sourceConfig': {
+                'config': {
+                    'type': 'DBT',
+                    'dbtConfigSource': dbt_config_source,
+                    'dbtUpdateDescriptions': True,
+                    'dbtUpdateOwners': True,
+                    'includeTags': True,
+                    'searchAcrossDatabases': False,
+                    'databaseFilterPattern': {
+                        'includes': ['.*']
+                    },
+                    'schemaFilterPattern': {
+                        'includes': ['.*']
+                    },
+                    'tableFilterPattern': {
+                        'includes': ['.*']
+                    }
```

Comment:
we should allow users to configure these values as well, we can make it optional and put default values if not provided